### ISSUE TITLE
CORS Proxy IO returning incorrect information

### ISSUE THREAD

**Issue by erwtj** [ikno-the-box-guy/IkSound#5]
Requests made using https://corsproxy.io/ as the proxy seem to return incorrect information from time to time. 

Request made to: ``https://corsproxy.io/?https://www.epidemicsound.com/json/tags/?sfx_tag=true``
```json
...
{
    "id": "4c264e01-303c-4bbb-8e82-71f727fc5777",
    "fatherGenreId": null,
    "tag": "jeongak",
    "displayTag": "Jeongak",
    "slug": "jeongak",
    "numTracks": 0
},
...
```

This happens both on the website as on something like postman. This could be a caching issue on corsproxy.io or epidemic, but could also be a regional difference. Not exactly sure what the cause of this is.

**Comment by erwtj** [comment]
Tried a proxy in US West, US East and EU and they didn't seem to have this issue

**Comment by erwtj** [comment]
Fixed by #6
